data/sql/updates/pending_db_world/rev_1688557801143854400.sql

```diff
@@ -0,0 +1,9 @@
+-- Vile Like Fire! (13071)
+-- fix Njorndar Proto-Drake vehicle unable to fly
+DELETE FROM `creature_template_spell` WHERE `CreatureID` = 30564;
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES (30564, 0, 57493, 12340);
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES (30564, 2, 7769, 12340);
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES (30564, 6, 57403, 12340);
+
```

Comment:
You could merge these 3 `insert into` in one line:
```suggestion
INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
(30564, 0, 57493, 12340),
(30564, 2, 7769, 12340),
(30564, 6, 57403, 12340);
```